File: matiasmasso/Mat.NET/DT2/DTOCce.vb
Public Class DTOCce
    Property Exercici As DTOExercici
    Property Cta As DTOPgcCta
    Property Amt As DTOAmt

    Public Sub New(oExercici As DTOExercici, oCta As DTOPgcCta)
        MyBase.New
        _Exercici = oExercici
        _Cta = oCta
    End Sub

    Public Sub New(oCta As DTOPgcCta, oAmt As DTOAmt)
        MyBase.New
        _Cta = oCta
        _Amt = oAmt
    End Sub
End Class

File: matiasmasso/Mat.NET/DTO/DTOCce.cs
namespace DTO
{
    public class DTOCce
    {
        public DTOExercici Exercici { get; set; }
        public DTOPgcCta Cta { get; set; }
        public DTOAmt Amt { get; set; }

        public DTOCce(DTOExercici oExercici, DTOPgcCta oCta) : base()
        {
            Exercici = oExercici;
            Cta = oCta;
        }

        public DTOCce(DTOPgcCta oCta, DTOAmt oAmt) : base()
        {
            Cta = oCta;
            Amt = oAmt;
        }
    }
}
